helpers.py

```diff
@@ -1,21 +1,15 @@
 import yaml
 import json
 import logging
-from definitions import ROOT_DIR
 
 def read_config(configpath):
     with open(configpath, 'r') as configfile:
-        config_data = json.load(configfile)
-        return config_data
-
-def read_clusterconf():
-    with open(f"{ROOT_DIR}/configs/cluster.yaml") as inputfile:
-        inputfile_info = yaml.load(inputfile, Loader=yaml.FullLoader)
-        return inputfile_info
-
-def read_passconfig():
```

Comment:
this wasn't used before or?

Reply:
Previously there were separate functions for each config file. Now we can use the read_config with all three (or more) configfiles